<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TKonstantinov" name="Command line test" tests="6" failures="2" timestamp="28 Oct 2013 12:41:58 GMT" time="19.224" errors="0">
  <testcase name="main" time="19.177" classname="test.test.rrrTest"/>
  <testcase name="main4" time="0.022" classname="test.test.rrrTest"/>
  <testcase name="main5" time="0.002" classname="test.test.rrrTest">
    <failure type="java.lang.AssertionError" message="This is forced error expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: This is forced error expected [false] but found [true]
	at test.test.rrrTest.main5(rrrTest.java:34)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- main5 -->
  <testcase name="main6" time="0.001" classname="test.test.rrrTest"/>
  <testcase name="main7" time="0.0" classname="test.test.rrrTest"/>
  <testcase name="main8" time="0.0" classname="test.test.rrrTest">
    <failure type="java.lang.AssertionError" message="This is forced error expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: This is forced error expected [true] but found [false]
	at test.test.rrrTest.main8(rrrTest.java:49)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- main8 -->
</testsuite> <!-- Command line test -->
